privacyidea/lib/tokens/smstoken.py

```diff
@@ -272,7 +272,7 @@ def create_challenge(self, transactionid=None, options=None):
         sms = ""
         options = options or {}
         return_message = "Enter the OTP from the SMS:"
-        attributes = {'state': transactionid}
+        attributes = {'state': transactionid, 'hideResponseInput': False}
```

Comment:
I think this is not the right place to do this. 
If you set it explicitly here to "False" you would have to do so in all other token libraries and this is obviously prone to errors.

I think we have to change it in the ``loginController.js``. (see below)